### PR TITLE
fix(ex): remove infura as rpc service

### PR DESCRIPTION
**Pull request by sam-goldman**

### CODE REVIEW

File: packages/executor/.env.example [hunk 1]
```diff
@@ -5,7 +5,5 @@ CHUGSPLASH_EXECUTOR__AMPLITUDE_KEY=<amplitude api key for analytics (optional)>
 CHUGSPLASH_EXECUTOR__LOG_LEVEL=<filter logs based on this level, i.e 'error', 'warning', 'info'>
 IPFS_PROJECT_ID=<ipfs project id to retrieve config file>
 IPFS_API_KEY_SECRET=<ipfs api key to retrieve config file>
-OPT_ETHERSCAN_API_KEY=<Optimism Etherscan API key>
-ETH_ETHERSCAN_API_KEY=<Ethereum Etherscan API key>
-INFURA_API_KEY=<Infura Project API Key>
```
Comment (sam-goldman) [inline packages/executor/.env.example:10]: Is it okay to remove `INFURA_API_KEY` here? It doesn't look like it's used in the executor package.

Comment (RPate97) [reply packages/executor/.env.example:10]: Yep good catch

File: packages/plugins/hardhat.config.ts [hunk 2]
```diff
@@ -22,17 +22,14 @@ const config: HardhatUserConfig = {
     },
   },
   networks: {
-    localhost: {
-      url: 'http://localhost:8545',
-    },
```
Comment (RPate97) [inline packages/plugins/hardhat.config.ts:27]: Why remove this?

**Review by RPate97 — APPROVED** [review]
Looks good

File: packages/plugins/hardhat.config.ts (hunk 2 above)
Comment (sam-goldman) [reply packages/plugins/hardhat.config.ts:27]: `http://localhost:8545` is the default port for the localhost Hardhat network, so there's no need to explicitly define this

**Pull request merged**